### ISSUE TITLE
Need a screen-reader styling method for words with wide letter-spacing.

### ISSUE THREAD

**Issue by cherscarlett** [gaearon/overreacted.io#403]
<img width="271" alt="Screen Shot 2019-03-18 at 3 32 37 PM" src="https://user-images.githubusercontent.com/260048/54568126-4fd6ad00-4993-11e9-95bf-b95bfd121841.png">

In a screen reader, these letters will be read one at a time. 

Solution would be to add a method to add letter-spacing via CSS, either by parsing during other operations, or with a specific method during entry.

**Comment by j-f1** [comment]
Is it possible to specify text to be read aloud instead of the actual text as an attribute? i.e. `<span aria-foo="aesthetics">a e s t h e t i c s</span>`

**Comment by cherscarlett** [comment]
@j-f1 You'd need to obscure the single letter text from the screen reader as well, otherwise it will read both. Because it's a style - it really should be CSS.

**Comment by hugmanrique** [comment]
I agree: we could use @j-f1's approach where the text is wrapped inside a component/span, but the text is **all together**, i.e. "aesthetics".

Then we could add `letter-spacing: 0.1 em` to that same component.
